frontend/src/container/ListOfDashboard/DashboardsList.tsx

```diff
@@ -1105,6 +1137,11 @@ export interface Data {
 	isLocked: boolean;
 	id: string;
 	image?: string;
+	widgets?: Array<WidgetRow | Widgets>;
+	layout?: Layout[];
+	panelMap?: Record<string, { widgets: Layout[]; collapsed: boolean }>;
+	variables: Record<string, IDashboardVariable>;
+	version?: string;
```

Comment:
Just verifying that this is the exhaustive list for data points that were missing ? there is no data point left here ?

Reply:
Yup. Tested as well after the changes (Exporting the dashboard and then uploading it back)